XamarinCommunityToolkit/Behaviors/MaxLengthValidationBehavior.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xamarin.Forms;
+
+namespace XamarinCommunityToolkit.Behaviors
+{
+    public class MaxLengthValidationBehavior : BaseBehavior<Entry>
```

Comment:
Only just realized this, but why is this one needed? `Entry` already has its own `MaxLength` property that does effectively the same and since we can only attach this one to an Entry... well... it seems obsolete 😅

Reply:
You could rewrite this to something similar to `NumericValidationBehavior` where it highlights the fact that you've gone over the character limit by changing textcolor but not actively clip it as that's what the existing property can be used for.

Reply:
I thought about your second point.  I did not delete it but instead changed MaxLength to ExceededLengthValidationBehavior and highlighted the text if MaxLength is exceeded.